Sources/KSuite/KSuiteGetProView.swift

```diff
@@ -0,0 +1,74 @@
+/*
+ iOS Features
+ Copyright (C) 2025 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import KSuiteUtils
+import SwiftUI
+
+public struct KSuiteGetProView: View {
+    public init() {}
+
+    public var body: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                HStack {
+                    Text(KSuiteLocalizable.kSuiteGetProTitle)
+                        .font(FontHelper.bodyMedium)
+
+                    KSuiteResources.badgePro.swiftUIImage
+                        .resizable()
+                        .scaledToFit()
```

Comment:
Why are we not using real Text(verbatim: "PRO") ?

Reply:
Because it is a custom font.